tests/test_clos/test_detect_workflow.py

```diff
@@ -0,0 +1,44 @@
+import mock
+import json
+import responses
+from responses import matchers
+from cloudos.clos import Cloudos
+
+INPUT = "tests/test_data/process_workflow_list_initial_request.json"
+APIKEY = 'vnoiweur89u2ongs'
+CLOUDOS_URL = 'http://cloudos.lifebit.ai'
+WORKSPACE_ID = 'lv89ufc838sdig'
+
+EXPECTED_WORKFLOWS_LIST = ["nextflow" , "docker" , "wdl"]
+
+@mock.patch('cloudos.clos', mock.MagicMock())
+@responses.activate
+def test_detect_workflow():
+    """
+    Test 'get_workflow_list' to work as intended
+    API request is mocked and replicated with json files
+    """
+    with open(INPUT) as json_data:
+        data = json.load(json_data)
+        json_data = json.dumps(data)
```

Comment:
```suggestion
    json_data = load_json_file(INPUT)
```
Why not to use the function in https://github.com/lifebit-ai/cloudos-cli/blob/main/tests/functions_for_pytest.py